src/Libraries/RevitNodes/Elements/Views/View.cs

```diff
@@ -160,21 +160,26 @@ public override string ToString()
         #region Filter
 
         /// <summary>
-        /// Add Filter to View
+        ///     Add Filter to View. Filter will be added even if View has a View Template applied, which normally would prevent user from adding
```

Comment:
done, but no guarantees that i will be able to consistently use proper grammar in my comments.